crates/anvil-polkadot/src/opts.rs

```diff
@@ -31,8 +31,10 @@ pub enum AnvilSubcommand {
     GenerateFigSpec,
 }
 
+pub struct SubstrateClient;
```

Comment:
SubstrateClient is not really accurate, let's call it SubstrateCli, just as the trait.
to avoid name clashing qualify the trait as sc_cli::SubstrateCli